### DISCUSSION TITLE
Can't set minDate in React-DatePicker

### DISCUSSION THREAD

**Discussion by CharlotteArai** [react-hook-form/react-hook-form#4276] [Q&A] [answered]
Hi!

So I am using react-datepicker in a react-hook-form. 
I can't figure out how to set a minDate together with a default value - whenever both these props are set, all dates are disabled in the calendar. 
If I clear the picker, it works again, but it would be nice if it worked without having to clear the field.

I've tried it outside of the form, and it works fine, so maybe I'm missing something about how I should do this in the form? 

I've made a sandbox illustrating the issue (I've used unix date, but I got the same result using new Date() as well).
https://codesandbox.io/s/react-hook-form-v6-controller-forked-u1rjm

Any help very much appreciated! 😀

**Comment by bluebill1049** [comment, marked as answer]
can you get it working with the `Controller`?

**Reply by CharlotteArai** [reply]
No, unfortunately not.  I'm using the Controller in the codesandbox and it's not working.

**Reply by bluebill1049** [reply]
sorry i mean without -__- typo

**Reply by CharlotteArai** [reply]
Yes, result is the same without the Controller.

**Reply by bluebill1049** [reply]
if that's the case then it's probably a good question for that lib?

**Reply by CharlotteArai** [reply]
Yup you are right, I thought it was only when I used the picker in the form the issue appeared but it's not. Thank you very much for your help and very quick reply! 🌻

**Reply by bluebill1049** [reply]
no worries :)

**Comment by leapful** [comment]
@CharlotteArai @bluebill1049 

There is nothing wrong react-datepicker or react-hook-form.

You are using min date is 25 Feb 2021 but selected date is 31 Mar 2020 therefore all days are before min date should be disabled.

**Reply by CharlotteArai** [reply]
Yes, I realized this as well. I got a fever, let's blame my confusion on that (I thought it was year 2020 now....) 🙃 Sorry to waste your time! And thanks for your input!

**Reply by bluebill1049** [reply]
Thank you @leapful 🙏
